### DISCUSSION TITLE
Development mode does not work anymore : no translations

### DISCUSSION THREAD

**Discussion by cnico** [home-assistant/frontend#14863] [Localization] [answered]
Hi, 

I would like to submit a PR in the frontend project.
I had my fork OK but when I synced my fork with the main frontend project on 17th december, all the translation files disappeared. And now when I run my fork locally, it runs with no label in the UI which makes it impossible to test my PR : 
![image](https://user-images.githubusercontent.com/6953620/209128902-40c10d0a-f0e0-4dbe-99a7-2e7d636aeb5d.png)

The github action from my fork also fails every day with this message : 
![image](https://user-images.githubusercontent.com/6953620/209128727-759ba5d4-688b-4e23-ba8f-eef9b8baac57.png)
You can see the details here : https://github.com/cnico/frontend/actions/runs/3757274985/jobs/6384296752

I checked the developper documentation but found nothing about it here : https://developers.home-assistant.io/docs/frontend/development

Could you tell me what I can do to run it properly localy ?
And if it is a temporary bug, is it know to you and will it be soon corrected ?

Regards,

**Comment by cnico** [comment]
Hi, I synced my fork and there is still the same problem after many commits, nobody developping on the frontend has this issue ?

**Comment by cnico** [comment, marked as answer]
I have found it. There is a new task in vs code which is named "Setup and fetch nightly translations". 
Running it resolves the problem...

**Reply by steverep** [reply]
Glad you found the task.  Translations were removed from the repository and instead downloaded on demand.  This keeps them much more up to date for developers and prevents contributors from mistakenly modifying them.  See #13381.

By default now though, only English is built unless you first setup a token to fetch them (since most developers don't need them).  Once you run the setup task, any future calls to develop or build tasks will update the translations if needed (unless you remove the token file).

Sorry for the confusion.  I still need to add this explanation to the developer docs.
